Makefile

```diff
@@ -334,8 +334,6 @@ ci-build-and-test:
 		airflow-dev-build \
 		jupyter-build \
 		peerscout-api-build \
-		notebook-lint \
```

Comment:
Can we keep `notebook-lint` please? I am terrible at keeping notebooks tidy (in particular unused imports), and that can help.

Reply:
ah sure.